facade/src/main/scala/react/virtuoso/GroupedVirtuoso.scala

```diff
@@ -38,7 +38,9 @@ final case class GroupedVirtuoso[D](
   groupContent:            js.UndefOr[Int => VdomNode] = js.undefined,
   groupCounts:             List[Int] = List(Int.MaxValue), // It would be cool if this was a NonEmptyList
   override val modifiers:  Seq[TagMod] = Seq.empty
-) extends GenericComponentPAC[GroupedVirtuoso.GroupedVirtuosoProps[D], GroupedVirtuoso[D]] {
+) extends GenericComponentPACF[GroupedVirtuoso.GroupedVirtuosoProps[D], GroupedVirtuoso[
+      D
+    ], VirtuosoComponent] {
```

Comment:
Ugh. Why does scalafmt do this?

Reply:
Ouch, I'll add a `format: off` comment.

Reply:
Maybe we can configure it not to split type parameter lists? However, I can think of cases where we do want that. Ideally, I guess, we want not to split type parameter lists where all of them are at most, say, 3 chars long?